components/sections/ProjectSection.jsx

```diff
@@ -0,0 +1,108 @@
+import React, { useLayoutEffect, useState } from "react";
+import Link from "next/link";
+import IMAGES from "../../data/projectImagesData";
+import Image from "next/image";
+import Wrapper from "../wrappers/Wrapper";
+
+const ProjectSections = () => {
+  const [link, setLink] = useState(0);
+  const [show, setShow] = useState(true);
+  useLayoutEffect(() => {
+    setShow(true);
+  }, [link]);
+
+  return (
+    <Wrapper
+      styles={`w-full desktop:mt-34 desktop:mb-31 tablet:mb-26.5 tablet:mt-27 ipad:mb-23 ipad:mt-17.2 mobile:mb-20.3 mobile:mt-7.06`}
+    >
+      <p className="text-secondary-100 text-2xl desktop:-space-y-6 desktop:mb-0 desktop:pl-1 mt-16 mb-4 ipad:mt-16 desktop:mt-0 leading-7">
+        Project
+      </p>
+
+      <div className="hidden desktop:grid desktop:grid-cols-9 relative mt-2.25">
+        <ul className="flex flex-col text-7.5xl cursor-pointer col-span-4 -space-y-5 leading-snug">
+          {IMAGES.map(
+            (item, i) =>
+              i !== 0 && (
+                <li
+                  key={i}
+                  className={`flex flex-col overflow-hidden desktop:inline-block mb-0`}
+                >
+                  <div
+                    className={` inline-block relative group desktop:after:content-[''] desktop:after:absolute desktop:after:h-0.5 desktop:after:w-0 desktop:after:bottom-5 desktop:after:hover:w-full desktop:after:left-0 desktop:after:origin-left desktop:after:bg-black desktop:hover:after:origin-left desktop:after:transition-all desktop:after:duration-200 desktop:after:ease-in-out desktop:after:delay-100`}
```

Comment:
```suggestion
                  className="flex flex-col overflow-hidden desktop:inline-block mb-0"
                >
                  <div
                    className="inline-block relative group desktop:after:content-[''] desktop:after:absolute desktop:after:h-0.5 desktop:after:w-0 desktop:after:bottom-5 desktop:after:hover:w-full desktop:after:left-0 desktop:after:origin-left desktop:after:bg-black desktop:hover:after:origin-left desktop:after:transition-all desktop:after:duration-200 desktop:after:ease-in-out desktop:after:delay-100"
```

Also here use BEM(export this long string as a single class in global.css)